migration-engine/migration-engine-tests/tests/migrations/foreign_keys.rs

```diff
@@ -284,7 +284,7 @@ fn changing_a_foreign_key_constrained_column_from_nullable_to_required_and_back_
     api.schema_push_w_datasource(dm).send().assert_green();
 }
 
-#[test_connector(exclude(CockroachDb), preview_features("referentialIntegrity"))]
+#[test_connector(exclude(CockroachDb, TiDB), preview_features("referentialIntegrity"))]
```

Comment:
Why does TiDB have to be excluded here? The tests is configured not to use foreign keys.
Is the problem again that the `User.id` primary key is renamed and changed (default removed)?

Reply:
Also because unsupported drop primary key when the table is using clustered index

update the comment on https://github.com/prisma/prisma-engines/commit/52d7328031bd4f31886b78244bfd2f801ccd708c